src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -148,6 +148,20 @@
                        IsHitTestVisible="False"
                        Background="{x:Null}"
                        Foreground="#555555">
+                        <TextBlock.Style>
+                            <Style TargetType="TextBlock">
+                                <Style.Triggers>
+                                    <DataTrigger Binding="{Binding UsingDefaultValue}" Value="True">
+                                        <Setter Property="FontStyle" Value="Italic"/>
+                                        <Setter Property="FontWeight" Value="Normal"/>
```

Comment:
do we need to explicitly set the font weight in these 2 places if it's not different from each other?